client/src/constants.js

```diff
@@ -136,6 +137,15 @@ export const appRoutes = [
     icon: MdGavel,
     active: false,
   },
+  {
+    name: 'Planning',
+    protected: true,
+    allowedRoles: rolesAvailable,
+    path: '/planning',
+    component: lazy(() => import('components/views/Planning')),
+    icon: MdInsertChart,
+    active: false,
```

Comment:
I believe Sujan removed this property `active` in the PR that just merged, so just update the branch and remove this line and should be good!

Reply:
sounds good!